field/composite.go

```diff
@@ -424,12 +426,20 @@ func (f *Composite) unpackSubfieldsByTag(data []byte) (int, error) {
 			tagBytes = f.spec.Tag.Pad.Unpad(tagBytes)
 		}
 		tag := string(tagBytes)
-		if _, ok := f.spec.Subfields[tag]; !ok {
+		if _, ok := f.spec.Subfields[tag]; !ok && !skipUnknownTLVTags(f) {
 			return 0, fmt.Errorf("failed to unpack subfield %v: field not defined in Spec", tag)
 		}
 
 		field, ok := f.subfields[tag]
 		if !ok {
+			// Obtain the length of the unknown tag and add it to the offset.
+			if skipUnknownTLVTags(f) {
+				fieldLength, readed, err := prefix.BerTLV.DecodeLength(999, data[offset:])
```

Comment:
First I wanted to ask if 999 is the max length, then I found that it's not used inside berTLV as length is read from data. I suggest to:
```suggestion
				fieldLength, readed, err := prefix.BerTLV.DecodeLength(ignoredMaxLen, data[offset:])
```

and define `ignoredMaxLen` before the method:

```go
// Because BER-TLV lengths are decoded dynamically, the maxLen method
// argument is ignored.
const ignoredMaxLen int = 0
```

Reply:
Great!
I didn't realize that maxLength is being ignored in `BerTLV` decoder. Now is implemented as you suggested.